mkdocs.yml

```diff
@@ -0,0 +1,19 @@
+site_name: Retis
+site_description: Documentation for Retis, the Linux packet tracing tool
+site_author: The Retis community
+site_url: https://retis-org.github.io/retis
```

Comment:
That url is there in case we want to publish with GH pages. It gets replaces with the actual readthedocs url.
Let me put it there as well to avoid any confusion.